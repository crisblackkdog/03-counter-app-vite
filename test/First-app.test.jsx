const { render } = require("@testing-library/react")
const { FirstApp } = require("../src/First-app")


describe('Pruebas First-app', () => { 
    test('debe hacer match con snapshot', () => { 

        const title ='Hola, soy goku';
        const {container} = render( <FirstApp title = {title} /> )
        // console.log(container);

        expect(container).toMatchSnapshot();
     })
    test('debe de mostrar el titulo en h1', () => { 

        const title ='Hola, soy goku';
        const {container, getByText} = render( <FirstApp title = {title} /> )

        expect(getByText(title)).toBeTruthy()

        const h1 = container.querySelector('h1')
        expect(h1.innerHTML).toContain(title)

     })
 })