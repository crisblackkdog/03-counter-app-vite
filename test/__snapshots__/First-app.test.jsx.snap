// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas First-app debe hacer match con snapshot 1`] = `
<div>
  <h1>
    Hola, soy goku
  </h1>
  <p>
    no hay subtitulo
  </p>
  <p>
    cristobal
  </p>
</div>
`;
